File: exercise/lotto/solution/wrong.java
import java.util.Comparator;
import java.util.HashSet;
import java.util.Set;
import java.util.concurrent.ThreadLocalRandom;
import java.util.stream.Collectors;

class Submission {

    public static String loterij(int aantal, int maximum) {
        ThreadLocalRandom r = ThreadLocalRandom.current();
        Set<Integer> result = new HashSet<>();
        while (result.size() < aantal) {
            result.add(r.nextInt(1, maximum + 1));
        }
        return result.stream()
                .sorted(Comparator.reverseOrder())
                .map(Object::toString)
                .collect(Collectors.joining(" - "));
    }

    public static String loterij(int aantal) {
        return loterij(aantal, 42);
    }

    public static String loterij() {
        return loterij(6, 42);
    }
}

File: exercise/lotto/solution/wrong.js
function loterij(aantal=6, maximum = 42) {
    const getallen = new Set();
    while (getallen.size < aantal) {
        getallen.add(Math.floor(Math.random() * maximum) + 1);
    }
    return [...getallen].sort((x, y) => y - x).join(" - ");
}
